resources/views/forum/_search.blade.php

```diff
@@ -66,7 +66,12 @@ class="forum-search-logo js-forum-search-button u-forum--bg-link"
                                 <i class="fa fa-angle-right"></i>
                             </button>
                         </form>
-                        {{ trans("forum.search.total_posts", ["posts_count" => $topic->postsCount()]) }}
+                        {{
+                            trans(
+                                "forum.search.total_posts",
+                                ["posts_count" => $topic->postsCount(priv_check('ForumTopicModerate')->can())]
```

Comment:
I don't think so. The only functionality it has has been replaced by the always visible footer bar.